electron/bridge.ts

```diff
@@ -1,6 +1,7 @@
 import { PaletteMode } from '@mui/material';
 import { contextBridge, ipcRenderer, IpcRendererEvent } from 'electron';
 import { promises as fs } from 'fs';
+import * as path from 'path';
```

Comment:
Can we just import sep rather than the whole module?

Reply:
Absolutely

Reply:
Done, sorry for that 🙈